KeeTrayTOTP/FormAbout.cs

```diff
@@ -156,5 +159,46 @@ private void FormAbout_FormClosed(object sender, FormClosedEventArgs e)
         {
             GlobalWindowManager.RemoveWindow(this);
         }
+
+        /// <summary>
+        /// Windows Form Mouse Click Handler.
+        /// </summary>
+        /// <param name="sender"></param>
+        /// <param name="e"></param>
+        private void ListViewAbout_MouseClick(object sender, MouseEventArgs e)
+        {
+            var hit = ListViewAbout.HitTest(e.Location);
+            if (hit.Item != null && hit.Item == ListViewAbout.Items[4] && hit.SubItem == ListViewAbout.Items[4].SubItems[1])
+            {
+                var url = new Uri(hit.SubItem.Text); 
+                System.Diagnostics.Process.Start(url.ToString());
```

Comment:
This can be simplified. The url is already available as ``Uri``. (see the static ``SupportUrl`` member)

``System.Diagnostics.Process.Start(hit.SubItem.Text);``
or
``System.Diagnostics.Process.Start(SupportUrl.ToString());``

Reply:
Thanks a lot for pointing these out!